server/install.go

```diff
@@ -403,7 +403,8 @@ func (ip *InstallationProcess) Execute() (string, error) {
 		AttachStdin:  true,
 		OpenStdin:    true,
 		Tty:          true,
-		Cmd:          []string{ip.Script.Entrypoint, "/mnt/install/install.sh"},
+		Entrypoint:   []string{ip.Script.Entrypoint, "/mnt/install/install.sh"},
```

Comment:
Why not set the entrypoint to `nil` and the command would stay the same?

Reply:
I tried that. It does not work.

Reply:
No. Just does not set it at all. I have to redo this as it brakes images with a cmd set.